algorithms/src/sorting/impl/Kokkos_SortImpl.hpp

```diff
@@ -399,9 +399,14 @@ sort_device_view_with_comparator(
 
   using ViewType = Kokkos::View<DataType, Properties...>;
   using MemSpace = typename ViewType::memory_space;
+// Note with HIP unified memory this code path is still the right thing to do
+// We don't have thrust in yet, and the create_mirror_view_and_copy will do
+// the right thing.
```

Comment:
We do have `rocthrust` support in `develop`.

Reply:
Yes, at the minimum the comment needs to be updated.

Reply:
Do you have a suggestion? I.e. are you expecting we never got reach this thing?